ui/imports/utils/Constants.qml

```diff
@@ -402,7 +402,7 @@ QtObject {
                 errorMessage: qsTr("Username must be at least 5 characters")
             },
             StatusRegularExpressionValidator {
-                regularExpression: /^[a-zA-Z0-9\-_]+$/
+                regularExpression: /^[a-zA-Z0-9\-_ ]+$/
                 errorMessage: qsTr("Only letters, numbers, underscores and hyphens allowed")
```

Comment:
Just realized... you might want to adjust this error message too (to include the possibility to contain space)

Reply:
Updated